bulk/sensor_bulk_load-AssetRecord.csv

```diff
@@ -396,12 +396,14 @@ CGINS-ADCPAM-43106,N00015,Sensor,1,ADCP Velocity Profiler (short range) for Glid
 CGINS-ADCPAM-43111,N00728,Sensor,1,ADCP Velocity Profiler (short range) for Glider: ADCPA Series M,ADCP for Slocum Glider,Teledyne RDI,Explorer DVL 600 kHz,643111,,20130125,,,
 CGINS-ADCPAM-43473,N00320,Sensor,1,ADCP Velocity Profiler (short range) for Glider: ADCPA Series M,ADCP for Slocum Glider,Teledyne RDI,Explorer DVL 600 kHz,643473,,20121228,,,
 CGINS-ADCPAM-43474,N00325,Sensor,1,ADCP Velocity Profiler (short range) for Glider: ADCPA Series M,ADCP for Slocum Glider,Teledyne RDI,Explorer DVL 600 kHz,643474,,20121228,,,
+CGINS-ADCPAM-44067,,Sensor,1,ADCP Velocity Profiler (short range) for Glider: ADCPA Series M,ADCP for Slocum Glider,Teledyne RDI,Explorer DVL 600 kHz,644067,,,,,

```

Comment:
I didn't see any documentation in Vault for this s/n.  Do we have any vendor documentation for it?

Reply:
Not that I could find. I'm still looking, but no luck so far. It was definitely deployed on 339 multiple times. We will need to create a calibration CSV for it as well.

Reply:
Roger that.  Maybe ping on vendor for some kind of documentation showing that this is actually a real s/n...

Reply:
Yeah we can try that. In the past they haven't had documentation relating to the xducer SN that we need. I was able to verify this is a real SN based on FCP results and the parsed recovered data. We have created "dummy" calibration CSVs in the past for glider DVLs when we don't have any documentation.

Reply:
I don't see this in the original list of 24 delivered for Coastal Pioneer & Endurance.  It must be a spare purchased later 2015.  I can dig for that